File: modul6/Python/PRAK602-2310817210023-RifkyPutraMahardika.py
jumlah_ruangan = int(input("Input :\nMasukkan jumlah ruangan :\n"))
limit = list(map(int, input("Banyaknya zetsu putih yang dimasukkan setiap ruangan :\n").split()))

print("\nOutput :\nZetsu putih setelah membelah diri :")
hasil = 0
for i in range(jumlah_ruangan):
    hasil = (i + 1) * limit[i]
    print(hasil, end=" ")

File: modul6/C/PRAK602-2310817210023-RifkyPutraMahardika.c
#include<stdio.h>

int main(){
    int limit[50];
    int i, j, jumlah_ruangan;
    printf("Input :\n");
    printf("Masukkan jumlah ruangan :\n");
    scanf("%d", &jumlah_ruangan);
    printf("Banyaknya zetsu putih yang dimasukkan setiap ruangan :\n");
    for(i=0; i<jumlah_ruangan; i++){
        scanf("%d", &limit[i]);
    }
    printf("\nOutput :\n");
    printf("Zetsu putih setelah membelah diri :\n");
    for(i=1; i<=jumlah_ruangan; i++){
        int hasil = 0;
        for(j=0; j<i; j++){
            hasil = i * limit[j];
        }
        printf("%d ", hasil);
    }
return 0;
}